hack/vmimage.sh

```diff
@@ -44,7 +44,7 @@ make vmimage-validate
 
 PHASE=image_tag
 TAGS=$(az image show -g "$IMAGE_RESOURCEGROUP" -n "$IMAGE_RESOURCENAME" --query tags | python -c 'import sys; import json; tags = json.load(sys.stdin); print " ".join(["%s=%s" % (k, v) for (k, v) in tags.items()])')
-az resource tag -g "$IMAGE_RESOURCEGROUP" -n $IMAGE_RESOURCENAME --resource-type Microsoft.Compute/images --tags $TAGS valid=true
+az resource tag -g "$IMAGE_RESOURCEGROUP" -n $IMAGE_RESOURCENAME --resource-type Microsoft.Compute/images --tags $TAGS valid=true $EXTRA_TAGS
```

Comment:
What is the format of `$EXTRA_TAGS`?

Reply:
`key=value` pairs, I am thinking of a use case where you tag an image as rc=true and the azure-purge bin doesn't delete it

Reply:
Thanks, :+1:

Reply:
why we adding this now if we dont use it?

Reply:
yeah you're right, reverting this part.